climada/entity/exposures/test/test_base.py

```diff
@@ -82,6 +82,13 @@ def test_assign_pass(self):
             self.assertEqual(exp.gdf.shape[0], len(exp.gdf[INDICATOR_CENTR + 'FL']))
             np.testing.assert_array_equal(exp.gdf[INDICATOR_CENTR + 'FL'].values, expected_result)
 
+    def test__init__meta_type(self):
+        """ Check if meta of type list raises a ValueError in __init__"""
+        with self.assertRaises(ValueError) as cm:
+            Exposures(None, meta=[], tag=None, ref_year=DEF_REF_YEAR,
+                 value_unit=DEF_VALUE_UNIT, crs=None)
```

Comment:
```suggestion
            Exposures(meta=[])
```
less is more (unless it's less) :wink;